managers/templates/managers/expected_hours_calendar.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
+{% load static %}
+
+{% load crispy_forms_tags %}
+{% load data_display_filters %}
+{% load data_structure_element_selectors %}
+
+{% block extra_head %}
+    <link
+        rel="stylesheet"
+        href="https://code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css"
+        integrity="sha384-Nlo8b0yiGl7Dn+BgLn4mxhIIBU6We7aeeiulNCjHdUv/eKHx59s3anfSUjExbDxn"
+        crossorigin="anonymous"
+    />
+    <link
+        rel="stylesheet"
+        type="text/css"
+        href="{% static 'employees/style.css' %}"
+        integrity="{% staticinline "employees/style.css" encode="sri" %}"
+        crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" href="{% static 'managers/vendor/caleandar.css' %}">
+
+{% endblock %}
+
+{% block content %}
+    <div class="container main-white-container margin-top-space">
+        {% include "common/popup_message.html" %}
+        {% include "employees/partial/modal_forms/modal_base.html" %}
+        {% include "employees/partial/month_navigation/report_list_navigation_bar.html" %}
+
+        <div id="project-select">
+            <b> Pick a project: </b>
+            <select id="project-select-options"></select>
+        </div>
+
+        <div id="employee-list">
+            <b> Pick employee: </b>
+            <div id="employee-list-options"></div>
+        </div>
+
+        <div id="calendar-feature"></div>
+    </div>
+{% endblock %}
+
+{% block extra_script %}
+    {{ month_form.media }}
+    <script type="text/javascript">  var curr_month = "{{ month }}"; </script>
+    <script type="text/javascript">  var curr_year = "{{ year }}"; </script>
+    <script src={% static 'managers/scripts/expected_hours_caleandar_ajax.js' %} defer></script>
+    <script src={% static 'managers/vendor/caleandar.js' %}></script>
```

Comment:
As far as I remember we have `staticinline` to use here instead of `static` to generate also integrity hashes

Reply:
Added